api/src/test/java/io/kaoto/backend/api/resource/v1/IntegrationsResourceTest.java

```diff
@@ -304,6 +306,7 @@ void scriptStep() throws Exception {
                 .statusCode(Response.Status.OK.getStatusCode());
 
         var flow = res.extract().body().as(Integration.class);
+        assertEquals(2, flow.getSteps().size());
```

Comment:
This equals are because on the first run these tests failed and I was wondering why... and well, another check is not bad.